targets/default/jerry-port-default.h

```diff
@@ -32,6 +34,8 @@ extern "C"
 void jerry_port_default_set_abort_on_fail (bool);
 bool jerry_port_default_is_abort_on_fail (void);
 
+void jerry_port_default_set_log_level (jerry_log_level_t);
```

Comment:
We have no use case for it but it would be nice to provide a getter as well, as a convenience.
```c
jerry_log_level_t jerry_port_default_set_log_level (void);
```